spec/factories/user.rb

```diff
@@ -1,27 +1,29 @@
-FactoryGirl.define do
-  factory :authentication_devise_user, :class => Refinery::Authentication::Devise::User do
-    sequence(:username) { |n| "refinery#{n}" }
-    sequence(:email) { |n| "refinery#{n}@example.com" }
-    password  "refinerycms"
-    password_confirmation "refinerycms"
-  end
+if defined?(Refinery::Authentication::Devise::User)
+  FactoryGirl.define do
+    factory :authentication_devise_user, :class => Refinery::Authentication::Devise::User do
```

Comment:
Normally, yes but long time ago i was not able to access to it, let me check if it's still relevant.